cmd/manager/main.go

```diff
@@ -266,30 +287,31 @@ func ensureDefaults(cfg *rest.Config, scheme *kruntime.Scheme) error {
 // cleanUpOldOpsrcResources cleans up old deployments and services associated with OperatorSources
 func cleanUpOldOpsrcResources(kubeClient client.Client) error {
 	ctx := context.TODO()
-	deploy := &v1.DeploymentList{}
-	svc := &v12.ServiceList{}
-	o := &client.ListOptions{}
-	if err := o.SetLabelSelector(strings.Join([]string{builders.OpsrcOwnerNameLabel, builders.OpsrcOwnerNamespaceLabel}, ",")); err != nil {
-		return err
+
+	deploy := &appsv1.DeploymentList{}
+	svc := &corev1.ServiceList{}
+	o := []client.ListOption{
+		client.MatchingLabels{builders.OpsrcOwnerNameLabel: builders.OpsrcOwnerNamespaceLabel},
```

Comment:
This is needed as there were breaking changes in the function signatures in the controller-runtime Client interface. The new signature expects a list of options vs. a single instance of a ListOption.